tests/authHelper.ts

```diff
@@ -0,0 +1,17 @@
+import { AuthManager, IAuthenticationInformation } from "poker/authmanager";
+
+export function getTestAuthManagerAuthenticated(login: string, loginId: number) {
```

Comment:
Instead of using AuthManager instance, better use fake instances like this
```
const notAuthenticated: IAuthenticationInformation = {
    login: () => null,
    loginId: () => null,
    authenticated: () => false,
}
```
This values could be placed in the test file which will be changed. Whole idea is to have very very simple implementation of interface which will behave exactly like we need. Mostly in tests authentication status does not changed and this would be sufficient implementation

Reply:
Now test object should include `registerAuthenticationChangedHandler` which do nothing